plugin/src/main/java/org/opensearch/ml/utils/MLExceptionUtils.java

```diff
@@ -24,6 +24,8 @@ public class MLExceptionUtils {
         "Local Model is currently disabled. To enable it, update the setting \"plugins.ml_commons.local_model.enabled\" to true.";
     public static final String AGENT_FRAMEWORK_DISABLED_ERR_MSG =
         "Agent Framework is currently disabled. To enable it, update the setting \"plugins.ml_commons.agent_framework_enabled\" to true.";
+    public static final String CONTROLLER_DISABLED_ERR_MSG =
+        "Controller is currently disabled. To enable it, update the setting \"plugins.ml_commons.controller_enabled\" to true.";
```

Comment:
Can we have more explicit message like "User level throttling controller" or something like that? Otherwise I'm not sure if customer knows what is Controller in general.

Reply:
The user must know what is controller in general because when a user calls the API it's `_controller` API. And in the future we will include fields more than user-level throttling in controller index, so, have message with user level throttling is not a maintainable way.